### PR TITLE
Merge from new builder node for eventState

### PR DESCRIPTION
**Pull request by hhou2011**
@d-shapiro

### CODE REVIEW

File: src/main/java/bacnet/LocalBacnetPoint.java [hunk 1]
```diff
@@ -373,23 +373,25 @@ protected void setupEventStateProperty(String name) {
...
 		propertyIdToLocalProperty.put(PropertyIdentifier.eventState, eventStateProperty);
 	}
-	
+
 	protected void setupPolarityProperty(String name) {
 		Node propertyNode = node.getChild(name);
 		NodeBuilder b = null;
 		if (null != propertyNode) {
-			
+
```
Comment (Pacane) [inline src/main/java/bacnet/LocalBacnetPoint.java:385]: just invert the if and remove this branch

File: src/main/java/bacnet/LocalBacnetPoint.java [hunk 2]
```diff
@@ -424,15 +426,22 @@ protected void setupOutOfServiceProperty(String name) {
...
 	private void setupRequiredProperty() {
 
 		ObjectType ot = objectId.getObjectType();
 
 		List<PropertyTypeDefinition> defs = ObjectProperties.getRequiredPropertyTypeDefinitions(ot);
 
 		for (PropertyTypeDefinition def : defs) {
-			// System.out.println(def.getClazz() + " : " +
-			// def.getPropertyIdentifier());
+			//System.out.println(def.getClazz() + " : " + def.getPropertyIdentifier());
```
Comment (Pacane) [inline src/main/java/bacnet/LocalBacnetPoint.java:444]: remove this if it's not used

File: src/main/java/bacnet/LocalBacnetPoint.java [hunk 3]
```diff
@@ -424,15 +426,22 @@ protected void setupOutOfServiceProperty(String name) {
 		propertyIdToLocalProperty.put(PropertyIdentifier.outOfService, outOfServiceProperty);
 	}
 
+	protected void setupPriorityArrayProperty(String name) {
+
```
Comment (Pacane) [inline src/main/java/bacnet/LocalBacnetPoint.java:430]: remove empty lines.

File: src/main/java/bacnet/LocalBacnetPoint.java [hunk 4]
```diff
@@ -373,23 +373,25 @@ protected void setupEventStateProperty(String name) {
 
 		LocalEventStateProperty eventStateProperty = new LocalEventStateProperty(objectId,
 				PropertyIdentifier.eventState, this, node, propertyNode);
-		if (b != null) b.build();
+		if (b != null)
```
Comment (Pacane) [inline src/main/java/bacnet/LocalBacnetPoint.java:376]: I'd add braces around this, sometimes it's a cause of errors.

**Comment by d-shapiro** [comment]
Closed, as commit does not appear to change anything and is behind the master branch

**Pull request closed without merging**